### ISSUE TITLE
Network Transport

### ISSUE THREAD

**Issue by t-mullen** [RationalCoding/realtime-collaboration-spec#2]
Anything used must be supported in both web-based editors and native editors.

I think WebRTC with a WebSocket fallback for non-web-based clients would work well. Implementing WebSockets would then be first priority.

**Comment by AGhost-7** [comment]
Question - do you have the intention of using webrtc with UDP?

**Comment by t-mullen** [comment]
No. The DataChannel for WebRTC uses SCTP, which is much like TCP. Were you worried about out-of-order delivery?

**Comment by AGhost-7** [comment]
Yea, I was thinking that some of the structures I've seen so far might not be suited for out of order messages.

**Comment by t-mullen** [comment]
I think TCP Websockets as a lowest-common-denominator is enough. Application developers can choose to support additional protocols.

WebRTC is too experimental to be used as the standard network transport.
